packages/core/services/PersistentConfigService/index.ts

```diff
@@ -31,6 +33,7 @@ export interface PersistedConfig {
     [PersistedConfigKeys.Queries]?: Query[];
     [PersistedConfigKeys.RecentAnnotations]?: string[];
     [PersistedConfigKeys.UserSelectedApplications]?: UserSelectedApplication[];
+    [PersistedConfigKeys.Environment]?: Environment;
```

Comment:
nit: alphabetical

Reply:
resolved in [b482dcf](https://github.com/AllenInstitute/biofile-finder/pull/296/commits/b482dcfc948a9bdd213afcb0021f679ba1bb6da8)